setup.py

```diff
@@ -35,19 +35,25 @@ def find_version(*file_paths):
           maintainer_email="psi29a@gmail.com",
           url="https://github.com/twisted/ldaptor",
           license="MIT",
+          python_requires="~=3.5",
           install_requires=[
               'passlib',
               'Twisted[tls]',
```

Comment:
the first version to support Python 3.5
```suggestion
              'Twisted[tls]>=15.5.0',
```

Reply:
@adiroiban do you want this change?